index.js

```diff
@@ -7,16 +7,9 @@ module.exports = {
   included: function(app) {
     this._super.included(app);
 
-    // Gridstack
-    ['gridstack', 'jquery-ui', 'gridstack.jQueryUI'].forEach(function(module) {
-      app.import({
-        development: 'node_modules/gridstack/dist/'+ module +'.js',
-        production:  'node_modules/gridstack/dist/'+ module +'.min.js'
-      });
-    });
-
-    app.import('node_modules/gridstack/dist/gridstack.css');
-    app.import('node_modules/gridstack/dist/gridstack-extra.css');
+    app.import("node_modules/gridstack/dist/gridstack.all.js");
```

Comment:
Could use single quotes, to be consistent with the rest of the code

Reply:
Had some issues with my formatter… Fixed now